<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationRGB</class>
 <widget class="QDialog" name="AnimationRGB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGB &amp; Animation Set Menu</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>29</y>
     <width>681</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSpinBox" name="spinBox">
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>RED</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="verticalSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>GREEN</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="verticalSlider_2">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>BLUE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_3">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="verticalSlider_3">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF TIME (ms):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ON TIME (ms):</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AnimationRGB</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AnimationRGB</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
